Mathlib/Algebra/GroupWithZero/Hom.lean

```diff
@@ -40,13 +40,6 @@ open Function
 namespace NeZero
 variable {F α β : Type*} [Zero α] [Zero β] [FunLike F α β] [ZeroHomClass F α β] {a : α}
 
-#adaptation_note
-/--
-We name `neZero` so it can be used as a named argument,
-but since https://github.com/leanprover/lean4/pull/5338, this is considered unused,
-so we need to disable the linter.
--/
-set_option linter.unusedVariables false in
 lemma of_map (f : F) [neZero : NeZero (f a)] : NeZero a :=
```

Comment:
Do you still need the name `neZero` for the instance argument? If not, how about this?
```suggestion
lemma of_map (f : F) [NeZero (f a)] : NeZero a :=

Reply:
I would prefer to keep this PR as simple as possible: I'm only removing notes that are outdated, and not changing any proper lean code.